frontend/src/types/SelectInputTypes.ts

```diff
@@ -0,0 +1,10 @@
+import { OptionBase } from "chakra-react-select";
+
+export interface Option extends OptionBase {
+  value: string | boolean;
+  label: string;
+}
```

Comment:
could you refactor `SelectFormInput` and `SelectFormInputClassroom` to use this interface as well? They have it defined separately in their individual files.

Reply:
I already changed it for `SelectFormInput` but will do so now for `SelectFormInputClassroom`!

Reply:
Or actually wait I take that back, SelectFormInputOption is slightly different since it can only have string as a value, so I could use MultiOption instead if that's cool

Reply:
Kind of the use case I create MultiOption for

Reply:
I have it already implemented for `SelectFormInput`, but for `SelectFormInputClassroom`, this type wouldn't work. An alternate for that file is the `MultiOption` type I defined, which covers the use case where values are only `string`, and not `string | boolean`.